CLC console documentation [MC-2390] (#287)
hazelcast/management-center-docs: docs/modules/tools/pages/console.adoc

= Executing Console Commands

> :description: The console feature enables you to execute client operations using built-in Hazelcast Command-Line Client (CLC).

Reviewer: ```suggestion
:description: The console feature allows you to execute client operations using the built-in Hazelcast Command-Line Client (CLC).
```

:page-aliases: monitor-imdg:console.adoc
:experimental: true

{description}

Management Center includes an interactive version of Hazelcast's Command-Line Client (CLC).
CLC functions as a client and adheres to any Security ACLs configured for it.

CLC offers support for a wide range of data structures and the ability to run SQL queries from a text terminal. 
It also provides flexible output options, such as table, CSV, and JSON formats.

> For more information on these features, visit: https://docs.hazelcast.com/clc/latest/overview

Reviewer: ```suggestion
For further information on these features, refer to the https://docs.hazelcast.com/clc/latest/overview[Overview^] topic of the CLC documentation.
```

. Open a console window by clicking on the **Console** button located at the top menu.
+
image:ROOT:Console.png[Console]

> . Execute any SQL query in terminal. You don't need to use any prefix for SQL queries.

Reviewer: ```suggestion
. Execute any SQL query in the terminal. Do not prefix SQL queries.
```

+
image:ROOT:ConsoleSql.png[Console SQL]

> . Execute CLC commands with `\` prefix, e.g. `\atomic-long`.

Reviewer: ```suggestion
. Prefix CLC commands with `\`. For example, `\atomic-long`.
```

+
image:ROOT:ConsoleCommand.png[Console Commands]

To get help or a list of all possible commands, enter `\help` in the console.

> You can use `--help` prefix with all commands to get more detail.

Reviewer: ```suggestion
You can use the `--help` option with any command to view the help for that command.
```

> TIP: Console container is resizable, you can change the height by dragging upper bar.

Reviewer: ```suggestion
TIP: The console container is resizable; to change the height, drag the top bar.
```

> You can make it fullscreen or minimized using the icons in the top-right corner of console container.

Reviewer: ```suggestion
You can maximize or minimize the console container using the icons in its top-right corner.
```
